packages/components/src/card/card/component.tsx

```diff
@@ -57,7 +57,11 @@ function UnconnectedCard(
 
 	return (
 		<ContextSystemProvider value={ contextProviderValue }>
```

Comment:
The `size` is still being passed into the context system here. Should it be removed? Maybe it should be kept for some future purpose where JS needs to know what the size is set to 🤔